dags/copy_deduplicate.py

```diff
@@ -229,10 +232,39 @@
         email=['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com'],
     )
 
+    # Daily and last seen views on top of every Glean application.
+
+    gcp_conn_id = "google_cloud_derived_datasets"
+    baseline_etl_kwargs = dict(
+        gcp_conn_id=gcp_conn_id,
+        project_id=GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id,
+        location="us-central1-a",
+        cluster_name="bq-load-gke-1",
+        namespace="default",
+        image="mozilla/bigquery-etl:latest",
+    )
+    baseline_args = [
+        "--project-id=moz-fx-data-shared-prod",
+        "--date={{ ds }}",
+        "--only=*_stable.baseline_v1"
```

Comment:
Is this how the `--only` option works? Is this the `only_tables` parameter or the `table_filter` parameter in `list_baseline_tables`?

https://github.com/mozilla/bigquery-etl/blob/aeccb5c5039a2671d9fe9a40adf69b44b22f73a4/bigquery_etl/glean_usage/common.py#L38-L49

Reply:
`--only` affects both what is passed as `only_tables` and what is passed as `table_filter`. The logic is fairly complex and you have to look at the `standard_args` implementation to see what all happens.